app/experimenter/openidc/tests/test_middleware.py

```diff
@@ -69,3 +69,45 @@ def test_user_created_with_correct_email_from_header(self):
 
         self.assertEqual(request.user.email, user_email)
         self.assertFalse(request.user.is_staff)
+
+    def test_dev_user_is_super_staff_user_when_debug_true(self):
+        dev_email = "dev@example.com"
+
+        self.assertEqual(dev_email, settings.DEV_USER_EMAIL)
```

Comment:
You don't need to do this, the test should pass regardless of what's actually in settings.py